model_wrappers.py

```diff
@@ -190,7 +320,7 @@ def _extract_argument_from_response(self, response: str) -> str:
 
 
 # meta-llama/Llama-2-7b-chat-hf, etc
-class Llama2Wrapper(HuggingFaceWrapper):
+class Llama2Wrapper(DebateFactory):
```

Comment:
This will inherit from_openai() ?

Reply:
Yeah the idea was that Llama models can be run with the OpenAI API if there is a local inference server like LMStudio. We don't necessarily need to support that but I thought it might be nice.